src/Tickets/Commerce/Settings.php

```diff
@@ -609,4 +609,22 @@ public static function is_licensed_plugin( $revalidate = false ) {
 
 		return $is_license_valid;
 	}
+
+	/**
+	 * Determine if free ticket is allowed in Tickets Commerce.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public static function is_free_ticket_allowed() {
+		/**
+		 * Filter to allow free tickets in Tickets Commerce.
+		 *
+		 * @since TBD
+		 *
+		 * @param bool $is_free_ticket_allowed Whether free tickets are allowed in Tickets Commerce.
+		 */
+		return apply_filters( 'tec_tickets_commerce_is_free_ticket_allowed', true );
+	}
```

Comment:
Would love to potentially create a `tec_tickets_commerce_is_free_tickets_allowed()` as a template that.

Reply:
Done